docs/www/tls-kubernetes.md

```diff
@@ -0,0 +1,375 @@
+```
+
+We’ll use the Kafka API to go through the TLS configuration, but you can use similar settings to set up TLS for the other APIs. For this guide, we assume external connectivity is disabled unless otherwise stated.
+
+The operator supports a single listener with TLS per API. Additionally, if you have two listeners for the same API and one of them has external connectivity enabled, the operator assumes that the latter listener has TLS enabled.
+
+Providing an issuer or certificate to the operator is only available for the Kafka API. 
+
+### Client Auth (mTLS)
+
+When you enable TLS, the client has the option of verifying the identity of the server, but to do this the client must have the CA’s certificate available in a trusted store.
+
+We can then enable client authentication (mTLS) on the same listener by setting `requireClientAuth` to `true`.
+
+So for example, we can edit our previous `yaml` file to add this new setting: 
+
+```yaml
+kafkaApi:
+    - port: 9092
+    tls:
+        enabled: true
+        requireClientAuth: true
+```
+
+When you enable client authentication, you must specify a key-pair (certificate and key). As we will later see, the operator can also generate the key-pair.
+
+When you finish setting up your configuration, apply it by running:
+
+```bash
+kubectl apply \
+-f <your-file-path.yaml>
+```
+
+Remember to change to the actual file path that you used. 
+
+
+## Managing Certificates
+
+### Certificates Creation
+
+The Redpanda operator uses [cert-manager](https://cert-manager.io) to generate certificates. 
+
+By default, when you enable TLS, the Redpanda operator generates a root certificate for each API and uses that to generate leaf certificates for the nodes and client. You can check what certificates were generated in the section down below. 
+
+Specifically for the Kafka API, the operator also offers two options: 
+
+1) provide an existing certificate issuer; 
+   
+2) provide a node certificate. 
+
+This functionality is only available for the Kafka API.
+
+To provide an existing certificate issuer we need to fill in the `issuerRef`.
+
+In the `yaml` configuration file add this section:
+
+```yaml
+    kafkaApi:
+    - port: 9092
+      tls:
+        enabled: true
+        issuerRef:
+          name: <issuer-name>
+          kind: ClusterIssuer
+```
+
+Similarly, to provide a particular certificate (as a secret) as the node certificate we need to fill in the `nodeSecretRef` field:
+
+```yaml
+    kafkaApi:
+    - port: 9092
+      tls:
+        enabled: true
+        nodeSecretRef:
+          name: cluster-tls-node-certificate
+          namespace: given-cert
+```
+
+If the secret is in a namespace different from the namespace of the Redpanda cluster, the operator copies it over.
+
+### Reading/Inspecting the certificates
+
+When enabling TLS, the Redpanda operator generates a root certificate for the Kafka API that is consecutively used to generate a “node” certificate for the Kafka API listener. 
```

Comment:
```suggestion
When you enable TLS, the Redpanda operator generates a root certificate for the Kafka API that is consecutively used to generate a “node” certificate for the Kafka API listener. 
```

2 questions - What does "consecutively" mean in this context, and why is "node" in quotes?

Reply:
The root certificate is used to generate the node certificate. @dimitriscruz is this correct?
"node" is in quotes because it's not exactly a node certificate. It's a certificate that's used by the node. But it's not a node itself. Does that make sense?

Reply:
I think the quotes are unclear. Can we spell it out?

Reply:
I think it's fine to remove the quotes, or at most write "generate a Redpanda node certificate", but I think it is assumed.

Reply:
Addressed.